esri2geojson/esri2geojson.py

```diff
@@ -25,18 +25,19 @@ def _collect_params(strings):
     
     return params
 
-def esri2geojson(esri_url, output_path, headers={}, params={}):
+def esri2geojson(esri_url, output_path, keep_sr, headers={}, params={}):
     ''' Convert single ESRI feature service URL to GeoJSON file.
     '''
     task = EsriRestDownloadTask('esri', params=params, headers=headers)
-    task.download(esri_url, output_path)
+    task.download(esri_url, output_path, keep_sr)
     
-parser = ArgumentParser(description='Convert single ESRI feature service URL to GeoJSON file.')
+parser = ArgumentParser(description='Convert single ESRI feature service URL to GeoJSON or CSV file.')
 
 parser.add_argument('esri_url', help='Required ESRI source URL.')
-parser.add_argument('geojson_path', help='Required output GeoJSON filename.')
+parser.add_argument('output_path', help='Required output filename.')
 
-parser.add_argument('-l', '--logfile', help='Optional log file name.')
+parser.add_argument('--keep_sr', help="Keep original projection",
```

Comment:
*Technically* GeoJSON [is required to be in epsg:4326](https://tools.ietf.org/html/rfc7946#section-4). 😄